src/containerapp/azext_containerapp/_archive_utils.py

```diff
@@ -186,6 +186,10 @@ def _load_dockerignore_file(source_location, original_docker_file_name):
 
 
 def _archive_file_recursively(tar, name, arcname, parent_ignored, parent_matching_rule_index, ignore_check):
+    if os.path.isfile(name) and arcname == "":
```

Comment:
What of arcname is `None`? `None == ""` return `false`

Reply:
I don't believe it could happen today given the logic, but I'm updating the code to also take "None" into account